src/messages/instance/types.ts

```diff
@@ -0,0 +1,20 @@
+import { BaseExecutableContent, ParentVolume, VolumePersistence } from "../types";
+
+/**
+ * Root file system of a VM instance.
+ * The root file system of an instance is built as a copy of a reference image, named parent image. The user determines a custom size and persistence model.
+ */
+export type RootfsVolume = {
+    parent: ParentVolume;
+    persistence: VolumePersistence;
+    size_mib: number; //Limit to 1 GiB
```

Comment:
Where is the size limit coming from?

Reply:
I saw it on the original message schema on the python lib here:
https://github.com/aleph-im/aleph-message/blob/main/aleph_message/models/execution/instance.py#L23C15-L23C38
https://github.com/aleph-im/aleph-message/blob/main/aleph_message/models/execution/volume.py#L62

Reply:
This limit has been increased to 100GB.

Reply:
https://github.com/aleph-im/aleph-message/blob/9458ec9084d3d5f6bbd5b0b5f83fc7e360ef1968/aleph_message/models/execution/volume.py#L62

Reply:
It's only a comment and not actually enforced in the SDK. Might be interesting to add this to the [constants](https://github.com/aleph-im/aleph-sdk-ts/blob/feature/support-instances/src/messages/constants.ts) file. (Along with the size threshold for a content to be inlined)

Reply:
The size threshold for inline messages is 200 KB.